response_operations_ui/templates/collection_exercise/ce-ci-section.html

```diff
@@ -8,7 +8,19 @@
           section. "
       })
     }}
+    {% set tblHeaders =
```

Comment:
Dina only wants it on EQ_AND_SEFT so this will need a slight tweak

Reply:
Adjusted code to only display on EQ_AND_SEFT surveys